webpack.config.js

```diff
@@ -48,4 +52,14 @@ module.exports = {
       },
     ],
   },
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, "src/tailwind.js"),
+          to: "[name][ext]",
+        },
+      ],
+    }),
+  ],
```

Comment:
do we need this for the fonts to work? 

cos i tot that we have it in tailwind.css can liao.

Reply:
this is a separate thing from fonts. it's the tw config that we export out. is to replace [this](https://github.com/TradeTrust/tradetrust-ui-components/pull/127/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L20)

Reply:
ohh ahhh!! ok can